docs/ops/state/savepoints.md

```diff
@@ -110,14 +110,11 @@ For example with a `FsStateBackend` or `RocksDBStateBackend`:
 /savepoints/savepoint-:shortjobid-:savepointid/...
 {% endhighlight %}
 
+After <a href="https://issues.apache.org/jira/browse/FLINK-5763">FLINK-5763</a> savepoint is self-contained and relocatable now, you can move the file and restore from any location.
 <div class="alert alert-info">
-  <strong>Note:</strong>
-Although it looks as if the savepoints may be moved, it is currently not possible due to absolute paths in the <code>_metadata</code> file.
-Please follow <a href="https://issues.apache.org/jira/browse/FLINK-5778">FLINK-5778</a> for progress on lifting this restriction.
+<strong>Note: </strong>savepoint relocatable does not support taskowned state(such as GenericWriteAhreadLog sink) and <a href="{% link ops/filesystems/s3.md %}#entropy-injection-for-s3-file-systems">entropy injection</a> currently.
```

Comment:
```suggestion
<strong>Note: </strong>relocatable savepoints currently do not support task-owned state(such as `GenericWriteAhreadLog` sink) or <a href="{% link ops/filesystems/s3.md %}#entropy-injection-for-s3-file-systems">entropy injection</a>.
```

Reply:
Can you actually add links to Jira tickets for these two points?

Reply:
@NicoK Do you mean adding the two Jira tickets which introduce these two features(FLINK-4939 for GenericWriteAheadSInk and FLINK-9061 for entropy-injection)?